portfolio/src/main/webapp/style.css

```diff
@@ -193,6 +347,38 @@ section.blue {
   clear: both;
```

Comment:
Is this still needed since you aren't using float anymore?

Reply:
yea nope! deleted